src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+public enum Colors {
```

Comment:
```suggestion
public enum Color {
```
enum naming convention in singular

Reply:
not fixed